### ISSUE TITLE
(URGENT) Failed in tag_repo job

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/mnconvert-ruby#7]
https://github.com/metanorma/mnconvert-ruby/runs/4690908249?check_suite_focus=true

The job failed on tag v1.14.

Ping @CAMOBAP

**Comment by CAMOBAP** [comment]
Fixed, release done
@ronaldtse BTW should we yank https://rubygems.org/gems/mnconvert/versions/2.0.0 ?

**Comment by ronaldtse** [comment]
@CAMOBAP yes please help yank. Thanks!

**Comment by CAMOBAP** [comment]
@ronaldtse I haven't permissions to do this:
```sh
gem yank mnconvert -v 2.0.0
Yanking gem from https://rubygems.org...
You do not have permission to delete this gem.
```

From another point we released some gem that depends on 2.0.0, so maybe we should not yank it

**Comment by ronaldtse** [comment]
@CAMOBAP I've added access for you to the gem, can you accept the invitation?

I think we should yank it since it becomes confusing for people.
